apis/cloud.redhat.com/v1alpha1/clowdjobinvocation_types.go

```diff
@@ -60,6 +60,12 @@ type ClowdJobInvocationSpec struct {
 
 	// Testing is the struct for building out test jobs (iqe, etc) in a CJI
 	Testing JobTestingSpec `json:"testing,omitempty"`
+
+	// invokes the job pods in "debug" mode -- i.e., command is swapped to the debugCommand
+	Debug bool `json:"debug,omitempty"`
+
+	// indicates pod command to run when in debug mode (default: "/bin/sh -c 'while true; do sleep 1; done'")
+	DebugCommand []string `json:"debugCommand,omitempty"`
```

Comment:
I'm still not sure if this is something we want to give people the flexibility to do in the CJI

Reply:
I like it given that we apparently can't even rely on `cat` lol. I think this should potentially be
```suggestion
	type DebugSpec struct {
	    Enabled bool `json:"enabled"`
	    Command []string `json:"debugCommand,omitempty"`
}
```